src/ck-core/init.C

```diff
@@ -1145,7 +1145,7 @@ static void _nullFn(void *, void *)
   CmiAbort("Null-Method Called. Program may have Unregistered Module!!\n");
 }
 
-extern void _registerLBDatabase(void);
+extern void _registerLBManager(void);
```

Comment:
Why doesn't the LBDatabase also need to be registered?

Reply:
There is an LBDatabase member object within LBManager and it gets initialized in LBManager's ctor.